.github/workflows/ci-app-cloud-e2e-test.yml

```diff
@@ -0,0 +1,132 @@
+name: cloud-e2e-testing
+
+# Used to run the e2e tests on lightning.ai
+on:
+  schedule:
```

Comment:
```suggestion
  workflow_dispatch: # TODO: add inputs for specific git_head

  schedule:
```

Reply:
Addressed by trigger on change in this file...